updatecli/updatecli.d/maven.yml

```diff
@@ -62,6 +62,15 @@ targets:
         keyword: "ARG"
         matcher: "MAVEN_VERSION"
     scmid: default
+  updateMavenJdk19NanoserverVersion:
+    name: Update the Maven version for the JDK17 container image
+    kind: dockerfile
+    spec:
+      file: maven/jdk19/Dockerfile.nanoserver
+      instruction:
+        keyword: "ARG"
+        matcher: "MAVEN_VERSION"
+    scmid: default
```

Comment:
This change should be done on the subsequent PR otherwise your checks will fail here

Reply:
you are correct: it does not fail (I though the file `maven/jdk19/Dockerfile.nanoserver` was added in this PR but no, it is already on the principal branch)